corehq/apps/users/static/users/js/custom_data_fields.js

```diff
@@ -44,7 +48,6 @@ hqDefine("users/js/custom_data_fields", [
         _.each(self.slugs, function (slug) {
             self[slug] = fieldModel({
                 value: options.user_data[slug] || originalProfileFields[slug],
-                disable: !!originalProfileFields[slug],
```

Comment:
Yes we do. But that is actually done [here](https://github.com/dimagi/commcare-hq/blob/dad9a4cf971361d51d67925b55ce61d6f6bb507e/corehq/apps/users/static/users/js/custom_data_fields.js#L79)

This deleted code wasn't originally doing anything because `fieldModel` hardcodes a value for `disable` [here](https://github.com/dimagi/commcare-hq/blob/75d62995b098c4d8cc346e58effc65f03d646175/corehq/apps/users/static/users/js/custom_data_fields.js#L15-L19). Brief related slack thread [here](https://dimagi.slack.com/archives/C0215NEFT8D/p1716966139221699)